src/main/resources/org/jenkinsci/plugins/github/GitHubPlugin/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.github.GitHubPlugin
+
+def st = namespace("jelly:stapler");
+
+set("instance", my.configuration);
```

Comment:
as i understand groovy allow you directly operate with variable, while jelly requires describe such actions. (i.e. like ant)